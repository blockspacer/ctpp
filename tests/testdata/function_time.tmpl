// Start

[<TMPL_var (time() > 1547549443)>]

// End :)
