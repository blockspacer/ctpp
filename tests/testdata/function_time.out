// Start

[1]

// End :)
